services/api/src/resources/backup/resolvers.ts

```diff
@@ -458,19 +461,32 @@ export const updateRestore: ResolverFn = async (
 export const getRestoreByBackupId: ResolverFn = async (
   { backupId },
   args,
-  { sqlClientPool }
+  { sqlClientPool },
+  info
 ) => {
   const rows = await query(
     sqlClientPool,
     Sql.selectRestoreByBackupId(backupId)
   );
   const row = R.prop(0, rows)
-  if (row && row.restoreLocation != null) {
+
+  if (!row || row.restoreLocation == null) {
+    return row;
+  }
+
+  const restoreLocationRequested = info.fieldNodes[0].selectionSet.selections.find(item => item.name.value === "restoreLocation");
```

Comment:
just checking - is there any way that this could come back null? I don't _think_ it would (thinking about the upstream graphQL stuff)

Reply:
I don't believe so, not unless something changes upstream.